config/document_type_selections.yml

```diff
@@ -28,17 +28,21 @@
       hostname: whitehall-admin
       path: /government/admin/speeches/new
 
-- id: guidance
+- id: guidance_title
```

Comment:
Ah, this is annoying. Thanks for persevering with a solution. I think we should try and use a different ID if possible, as "title" suggests it's not an ID. How about "guidance_publications"?